prototype/src/main/webapp/script.js

```diff
@@ -0,0 +1,39 @@
+PROPOSED_TIMES_NO = 3; // Initially we hardcoded 3 different times suggestions
+
+function voteTime() {
+  // get the answers from the form and update votes on screen
+  // NOTE this data would be sent to a servlet in MVP and the data fetched to update UI
+  let inputs = document.getElementById("vote-time").elements;
+  for (let i = 0; i < PROPOSED_TIMES_NO; i++) {
+    if (inputs[i].checked === true) {
+      document.getElementById(inputs[i].id + "-vote").innerText = (
```

Comment:
I would make `document.getElementById(inputs[i].id + "-vote")` into it's own variable as it makes it clearer that the code is incrementing the element's value by one.

Reply:
Okay, changed it!